helm/examples/azure/sa/README.md

```diff
@@ -0,0 +1,70 @@
+# HPCC Azure storage with a Storage Account
+
+## Directory contents
+
+### Introduction
+Persistent storage can be created manually or through Kubernetes Clusters. If created through Kubernetes the storage will be gone when Persistent Volume (PV) or Kubernetes clusters are deleted. For Azure manually created shared storage with a storage account will not depend on any Kubernetes cluster. The data generated will be persistent and re-used as long the PV/PVC names and mount points are the same.
+
+### Create a share with a storage account
+If you don't have a storage account and a shared volume you can create them. If you already have them, provide the information in env-sa and ../hpcc-azurefile/values.yaml and generate a Kuernetes secret, if  you do not have one yet."
+To create a shared storage including storage account you can reference Reference: https://docs.microsoft.com/en-us/azure/aks/azure-files-volume
+We provide a set of scripts to help with the process. Before you run any script make sure to login to Azure first:
+```console
+az login
+```
+
+### Environment variables
+env-sa defines some variables for creating a storage account and shared storage.
+Some variables have default values but some of them don't and you must provide the values:
+- SUBSCRIPTION: Azure subscription
+- STORAGE_ACCOUNT_NAME: Storage Account name which will be generated
+- SA_RESOURCE_GROUP: a resource group name which must be unique in your subscription
+- SA_LOCATION: the region the storage account will be created in. It is recommended to use the same region as the Kubernetes cluster.
```

Comment:
Not sure about use of this file (env-sa) to defined (some but not all) of the required settings.
Might be okay as an alternative, but should be able to supply options on the command line to these scripts.

Reply:
Yes, there are two part of settings: values.yaml and env-sa:
1) Create the PersistentVolume and PersistentVolumeClaim: using values.yaml only
2) Create a storage account and a share (create-sa.sh): using values.yaml and env-sa.  This, I expect, users normally will do only once if they don't have the storage account. I will an option to create a share volume with existing storage account
3) Create a Kubernetes secret: using values.yaml and env-sa. This is only need once per Kubernetes cluster.